dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Protections/BaseClasses.mo

```diff
@@ -0,0 +1,207 @@
+within Dynawo.Electrical.Controls.Protections;
+
+/*
+* Copyright (c) 2022, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools for power systems.
+*/
+
+package BaseClasses "Base models for protections"
+  extends Icons.BasesPackage;
+
+  partial model BaseDistanceProtectionLine "Base model for line distance protection"
+    import Modelica;
```

Comment:
no need to import Modelica anymore. All essential imports are added in Dynawo package